agdb_api/typescript/src/client.ts

```diff
@@ -6,18 +6,34 @@ export class AgdbApi {
     private static c: Client | undefined = undefined;
     private static token: string = "";
 
-    static async client(host: String, port: number): Promise<Client> {
+    static async client(
+        host: String,
+        port: number,
+        publicPath?: string,
+    ): Promise<Client> {
         if (AgdbApi.c === undefined) {
-            AgdbApi.api = new OpenAPIClientAxios({
-                definition: `${host}:${port}/api/v1/openapi.json`,
-            });
-            AgdbApi.c = await AgdbApi.api.init<Client>();
-            AgdbApi.c.interceptors.request.use((config) => {
-                if (AgdbApi.token !== "") {
-                    config.headers.Authorization = `Bearer ${AgdbApi.token}`;
+            try {
+                let baseurl_tmp = `${host}:${port}`;
+                if (typeof publicPath !== "undefined") {
+                    baseurl_tmp = baseurl_tmp + `/${publicPath}`;
                 }
-                return config;
-            });
+                const baseURL = baseurl_tmp;
```

Comment:
Why repeat it to another variable? You could use `baseURL` from the start. Also the names should be either `mixedCase` or `snake_case`, not a mix. :-)

Reply:
I think this PR should be merged with the other one. The benefit of the "monorepo" is that relůated changes to all components can be done in one go.

Also you should add support for base path to the Rust client as well if you can. If not I will add it either as part of the PR or separately. Thanks!